### PR TITLE
chore(mojaloop/csi-190): update enums for proxy transfer states

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -259,6 +259,7 @@ declare namespace CentralServicesShared {
     ORACLE = 'oracle',
     POSITION = 'position',
     PREPARE = 'prepare',
+    PREPARE_FORWARDED = 'prepare-forwarded',
```
Comment (vijayg10) [inline src/index.d.ts:262]: Doesn't it need to be in action Enum?

File: src/enums/events.js [hunk 2]
```diff
@@ -101,6 +101,7 @@ const Event = {
     POSITION_PREPARE: 'position-prepare',
     POSITION_FULFIL: 'position-fulfil',
     PREPARE: 'prepare',
+    PREPARE_FORWARDED: 'prepare-forwarded',
```
Comment (vijayg10) [inline src/enums/events.js:104]: Do we need to call it prepare-forwarded? I think when we use this with event 'prepare', may be we can just call it 'forwarded'. So Event is `prepare` and action is `forwarded`

File: src/index.d.ts (hunk 1 above)
Comment (kleyow) [reply src/index.d.ts:262]: Confused. It's declared in the Event.Action part of the dict.

Comment (kleyow) [reply src/index.d.ts:262]: Oh nvm, index.d.ts file. done.

File: src/enums/events.js (hunk 2 above)
Comment (kleyow) [reply src/enums/events.js]: Sounds good. done.

**Pull request merged**